nesh-dev/unsullied-DRF
# #162758465 add tests to get one and many tasks

## What does this PR do?
  Add tests to get one and many tasks

## Description of tasks to be completed
* test models to ensure the data expected is valid
* test the response to ensure it is as expected
* test for all tasks
* tests a single task

## How should this be manually tested?
After cloning the repo, pull the branch ch-tests-162758465
* Create a virtual environment and activate it:
      ``` python3 -m virtualenv env ```
      ``` source env/bin/activate ```
* Install the dependencies ``` pip install -r requirements.txt ```

* Make and migrate migrations while in the same level as the manage.py file
   ```
         ./manage.py makemigrations
          ./manage.py migrate

   ```
  
* Runserver ``` ./manage.py test ```

* the result should be OK 

### What are the relevant Pivotal Tracker stories?
#162758465
